evaluate/metrics/testing/assessments.go

```diff
@@ -17,17 +19,48 @@ func AssertAssessmentsEqual(t *testing.T, expected metrics.Assessments, actual m
 	expected = maps.Clone(expected)
 	actual = maps.Clone(actual)
 
-	expected[metrics.AssessmentKeyProcessingTime] = 0
-	actual[metrics.AssessmentKeyProcessingTime] = 0
+	ignoreNonDeterministicAssessmentValues(expected)
+	ignoreNonDeterministicAssessmentValues(actual)
+	// expected[metrics.AssessmentKeyProcessingTime] = 0
+	// actual[metrics.AssessmentKeyProcessingTime] = 0
 
-	expected[metrics.AssessmentKeyGenerateTestsForFileCharacterCount] = 0
-	actual[metrics.AssessmentKeyGenerateTestsForFileCharacterCount] = 0
-	expected[metrics.AssessmentKeyResponseCharacterCount] = 0
-	actual[metrics.AssessmentKeyResponseCharacterCount] = 0
+	// expected[metrics.AssessmentKeyGenerateTestsForFileCharacterCount] = 0
+	// actual[metrics.AssessmentKeyGenerateTestsForFileCharacterCount] = 0
+	// expected[metrics.AssessmentKeyResponseCharacterCount] = 0
+	// actual[metrics.AssessmentKeyResponseCharacterCount] = 0
 
```

Comment:
errrrrr no comments in code that we want to merge

Reply:
Missed that, I thought I have deleted those